src/data/shared/case-studies/case-studies-list.js

```diff
@@ -385,24 +486,111 @@ const caseStudiesData = {
         },
       ],
     },
+    {
+      title: 'Aqua Security',
+      logo: Aqua,
+      description:
+        '<strong>Aqua Security</strong> uses eBPF to handle runtime detection and forensics',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://youtu.be/BF42-cl-8B0?si=FM6Mp2PfewzX7BnS',
+        },
+      ],
+    },
     {
       title: 'Akita',
       logo: Akita,
-      description: '<strong>Akita</strong> uses eBPF at scale for API observability.',
+      description: '<strong>Akita</strong> uses eBPF at scale for API observability',
       links: [
         {
           text: 'Docs',
           to: 'https://docs.akita.software/docs/how-akita-works#what-makes-akita-different',
         },
       ],
     },
+    {
+      title: 'Line Coporation',
+      logo: LineCoporation,
+      description:
+        '<strong>Line Corporation</strong> uses eBPF at scale for load balancing and tracing in their private cloud',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://www.youtube.com/watch?v=cxfVpBYlol4',
+        },
+      ],
+    },
+    {
+      title: 'Exein',
+      logo: Exein,
+      description:
+        '<strong>Exein</strong> uses eBPF in their open source runtime security framework for IoT',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://www.youtube.com/watch?v=vmRQXRit-sY&list=PLDg_GiBbAx-mAwr1P5H4xFEn7X-qWZ2V_&index=16',
+        },
+      ],
+    },
+    {
+      title: 'Ant Group',
+      logo: AntGroup,
+      description:
+        '<strong>Ant Group</strong> uses eBPF in their cloud native security paltform for vulnerability live-patching, and system anomaly detection',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://youtu.be/TXlxgDvxcoY?si=lvC1HqNl4jXNyfD0',
+        },
+      ],
+    },
+    {
+      title: 'Sysdig',
+      logo: Sysdig,
+      description:
+        '<strong>Sysdig</strong> uses eBPF to enhance the performance of Falco, their open-source runtime security project',
```

Comment:
Falco isn't their project, its in the CNCF now

Reply:
But they still actively maintain it right? Should I remove them?